client/payment-methods-map.tsx

```diff
@@ -247,6 +248,37 @@ const PaymentMethodInformationObject: Record<
 			'woocommerce-payments'
 		),
 	},
+	klarna: {
+		id: 'klarna',
+		label: __( 'Klarna', 'woocommerce-payments' ),
+		brandTitles: {
+			affirm: __( 'Klarna', 'woocommerce-payments' ),
+		},
+		description: __(
+			// translators: %s is the store currency.
+			// TODO: Is "Available to all customers paying in %s." correct for BNPLs given the introduction of domestic check for PMs?.
+			'Allow customers to pay over time with Klarna. Available to all customers paying in %s.',
+			'woocommerce-payments'
+		),
+		icon: iconComponent( KlarnaIcon, 'Klarna' ),
+		currencies: [
```

Comment:
This is probably is outdated now.

Reply:
Yes, good catch, thank you. Another example of why need that "one source-of-truth" implemenation